locales/ptBR.json

```diff
@@ -0,0 +1 @@
ing files":"Bestanden uploaden","Used for programmatic access to the fieldset.":"Used for programmatic access to the fieldset.","User created":"User created","User name":"User name","Username":"Username","Users":"Users","Users and groups settings":"Users and groups settings","Version Overview":"Version Overview","View":"Bekijken","View changes":"Bekijk wijzigingen","View this revision":"Bekijk deze revisie","Viewmode":"Viewmode","We apologize for the inconvenience, but the page you were trying to access is not at this address. You can use the links below to help you find what you are looking for.":"Onze excuses voor het ongemak, maar de pagina die u probeerde te bereiken bestaat niet. U kunt onderstaande links gebruiken om proberen te vinden wat u zocht.","We will use this address if you need to recover your password":"Wij gebruiken dit adres wanneer u uw wachtwoord wilt terugkrijgen","What":"Wat","When":"Wanneer","When this date is reached, the content will nolonger be visible in listings and searches.":"Wanneer deze datum bereikt is zal de inhoud niet langer zichtbaar zijn in lijsten en zoekresultaten.","Who":"Wie","Workflow updated.":"Workflow updated.","You can add a comment by filling out the form below. Plain text formatting.":"You can add a comment by filling out the form below. Plain text formatting.","You can control who can view and edit your item using the list below.":"U kunt in de onderstaande lijst beheren wie de content kan bekijken en beheren.","You can view the difference of the revisions below.":"U kunt de verschillen tussen de revisies hieronder bekijken.","You can view the history of your item below.":"U kunt de geschiedenis van uw item hieronder bekijken.","Your email is required for reset your password.":"Your email is required for reset your password.","Your location - either city and country - or in a company setting, where your office is located.":"Uw woonplaats - stad en land - of als u vanuit een bedrijf werkt, waar uw kantoor is gevestigd.","Your password has been set successfully. You may now {link} with your new password.":"Your password has been set successfully. You may now {link} with your new password.","Your preferred language":"De taal van uw voorkeur","box_forgot_password_option":"box_forgot_password_option","common":"common","description":"description","description_lost_password":"description_lost_password","description_sent_password":"description_sent_password","heading_sent_password":"heading_sent_password","hero":"hero","html":"html","image":"image","label_my_email_address_is":"label_my_email_address_is","maps":"maps","media":"media","mostUsed":"mostUsed","registration form":"registration form","table":"table","text":"text","title":"title","video":"video","we can send you a new one":"we can send you a new one","{count} selected":"{count} geselecteerd","{title} copied.":"{title} gekopieerd.","{title} cut.":"{title} geknipt.","{title} has been deleted.":"{title} is verwijderd.","{user} says:":"{user} says:"}
```

Comment:
@LeuAlmeida You only needed to rename the **"locales/ptBR.json"** file, but since you additionally added the **"locales/pt_BR.json"** file then delete the **"locales/ptBR.json"** file and leave only the **"locales/pt_BR.json"** file

Reply:
I'm sorry Leo, as I changed the folder name and generated a new json file I just forgot to delete the old file.
It is?